src/components/pages/__test__/HomePage.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import HomePage from '../HomePage'
+import configureMockStore from 'redux-mock-store'
+import { BrowserRouter, Route } from 'react-router-dom';
+
+let props, mountedComponent, store;
+
+const mockStore = configureMockStore();
+const homePage = () => {
+  if (!mountedComponent) {
+    mountedComponent = shallow(
```

Comment:
We should do a "mount" here instead of a shallow.  

To get around the problem of the "mount" throwing exceptions because it expects a router, I discovered a MockRouter you can import and use instead, see:

https://github.com/openmrs/openmrs-react-components/blob/master/src/components/routes/__tests__/AuthenticatedRoute.test.jsx#L2

https://github.com/openmrs/openmrs-react-components/blob/master/src/components/routes/__tests__/AuthenticatedRoute.test.jsx#L27

You'll need to react-mock-router as a devDependency of the project.